editor/js/Sidebar.Settings.Viewport.js

```diff
@@ -12,8 +12,32 @@ Sidebar.Settings.Viewport = function ( editor ) {
 
 	container.add( new UI.Text( strings.getKey( 'sidebar/settings/viewport/grid' ) ).setWidth( '90px' ) );
 
+	var div = new UI.Div().setPaddingLeft( '90px' );
+	container.add( div );
+
+	div.add( new UI.Text( 'Visible' ).setPaddingRight( '8px' ).setWidth( '90px' ) );
 	var show = new UI.THREE.Boolean( true ).onChange( update );
-	container.add( show );
+	div.add( show, new UI.Break() );
+
+	div.add( new UI.Text( 'Size' ).setPaddingRight( '8px' ).setWidth( '90px' ) );
+	var size = new UI.Integer( 30 ).setWidth( '45px' ).onChange( update );
```

Comment:
It's a bit unfortunate that you can set a value smaller than 0. Same for divisions and dividers:

![image](https://user-images.githubusercontent.com/12612165/53032144-88579980-346e-11e9-82c4-09ca2ca87ff3.png)


I think it's better to allow just positive integer values.